samples/snippets/core/system-text-json/csharp/Utf8ReaderPartialRead.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.IO;
+using System.Text.Json;
+
+namespace SystemTextJsonSamples
+{
+    public class Utf8ReaderPartialRead
+    {
+        public static void Run()
+        {
+            var jsonString = @"{
+                ""Date"": ""2019-08-01T00:00:00-07:00"",
+                ""Temperature"": 25,
+                ""TemperatureRanges"": {
+                    ""Cold"": { ""High"": 20, ""Low"": -10 },
+                    ""Hot"": { ""High"": 60, ""Low"": 20 }
+                },
+                ""Summary"": ""Hot"",
+            }";
+
+            var bytes = System.Text.Encoding.UTF8.GetBytes(jsonString.Trim());
```

Comment:
Trimming isn't necessary. Remove it.
```suggestion
            byte[] bytes = Encoding.UTF8.GetBytes(jsonString);
```